WowPacketParser/SQL/Builders/Spawns.cs

```diff
@@ -191,7 +201,7 @@ public static string Creature(Dictionary<WowGuid, Unit> units)
                     addonRows.Add(addonRow);
                 }
 
-                if (creature.IsTemporarySpawn())
+                if (creature.IsTemporarySpawn() && !Settings.SaveTempSpawns)
```

Comment:
Exporting temporary spawns for creatures won't actually work with just this change, because they are being excluded from the Unit list that is passed to the builder methods here.

https://github.com/TrinityCore/WowPacketParser/blob/master/WowPacketParser/SQL/Builder.cs#L78

Excluding those creatures from the list there is bad, because it excludes them from all tables, not just the `creature` spawns table. You won't get any `creature_template` data for them either, unless there was at least one permanent spawn of this creature id.